packages/core-snapshots-cli/lib/db/queries/rounds/delete-from-round.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM ROUNDS
```

Comment:
Properties in queries should be all lower-case, only SQL keywords should be uppercase.